src/sections/BlockInfoSection.tsx

```diff
@@ -210,19 +210,18 @@ const TransactionRow: FC<TransactionRowProps> = ({ transaction }) => {
           <TightLink to={`/transactions/${t.hash}`} text={t.hash} maxWidth="150px" />
         </td>
         <td>
-          {t.inputs.length} address{t.inputs.length > 1 ? 'es' : ''}
+          {t.inputs ? t.inputs.length : 0} {t.inputs && t.inputs.length === 1 ? 'address' : 'addresses'}
         </td>
         <td>
           <ArrowRight size={15} />
         </td>
         <td>
-          {t.outputs.length} address{t.outputs.length > 1 ? 'es' : ''}
+          {t.outputs ? t.outputs.length : 0} {t.outputs && t.outputs.length === 1 ? 'address' : 'address'}
```

Comment:
address**es**

Reply:
- 0 addresses
- 1 address
- 2 addresses

Reply:
- _"I have 0 dollars in my wallet"_
- _"I have 1 dollar in my wallet"_
- _"I have 2 dollars in my wallet"_

Reply:
mhh yes? Please look at your code.

Reply:
oops-a-daisy, sometimes I speak before I think, sorry!